Gemfile.lock

```diff
@@ -443,5 +443,8 @@ DEPENDENCIES
   webmock
   xray-rails
 
+RUBY VERSION
```

Comment:
No worries. Everything else looks great. Super glad to have your first PR and welcome to Operation Code!